user-console/src/main/java/org/pentaho/mantle/client/commands/DeleteFolderCommand.java

```diff
@@ -133,6 +134,7 @@ public void onResponseReceived( Request request, Response response ) {
                 dialogBox.center();
                 event.setMessage( Messages.getString( "couldNotDeleteFolder" ) );
                 EventBusUtil.EVENT_BUS.fireEvent( event );
+                dialogBox.hide();
```

Comment:
Hiding the dialogBox immediately after showing it (using `center()`)?
Why is this necessary?

Reply:
Removing the line causes the dialog to remain on the screen even after pressing the "OK" button.

Reply:
But this way, the dialog would not even be shown...

Reply:
After rebasing I have observed that the focus is set to `No` instead of `Yes, Move to Trash` upon opening the dialog, also the error dialog is now taking two keyboard/mouse inputs to close.
.

Reply:
@dcleao can you look into this? Thank you.

Reply:
The "No" button being selected by default is the new default, as it is the safest option.

It's arguable that, for this dialog, the default button could be "Yes, Move to Trash", given moving to trash is a reversible action.

If desired, the default can be changed, by using the dialog's `setFocusWidget( . )` method. However, the `okButton` field is protected, and as such, it is currently necessary to create a subclass (it can be an anonymous one).

Reply:
I'm not sure what you mean by "error dialog"? This dialog is a confirmation dialog. If you are referring to it, when the default is "No", accepting the action will require two keyboard inputs: a Tab and an Enter. This is expected, if we agree that the default button should be "No".